### PR TITLE
PAE-849 Update Pathways footer

### PR DESCRIPTION
**Pull request by anfbermudezme**
The objective of this PR is update the footer of pearson-pathways-theme, following the guidelines of figma:

https://pearsonadvance.atlassian.net/browse/PAE-849, 
https://www.figma.com/file/xShZyCR6BCyvmtmJ7pZVjc/Intermidiate-DS?node-id=347%3A8677

The footer before the changes is:

Selection_193

The footer after the changes is:

Selection_194

### CODE REVIEW

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 1]
```diff
@@ -34,13 +35,20 @@ $main-font-size: 0.875em;
       }
 
       h4 {
-        font-family: $serif;
-        font-size: 1.25em;
+        font-family: 'Open Sans';
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Change this to a variable definition. @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 2]
```diff
@@ -34,13 +35,20 @@ $main-font-size: 0.875em;
...
+        font-style: 'normal';
+        font-weight: 600;
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Are these variables necessary? @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 3]
```diff
@@ -34,13 +35,20 @@ $main-font-size: 0.875em;
...
+        font-size: 18px;
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Use relative units instead of absolute units. @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 4]
```diff
@@ -50,7 +58,8 @@ $main-font-size: 0.875em;
         margin: 0 14px 0 0;
 
         a {
-          color: $footer-contrast;
+          height: 60px;
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Is this necessary? @anfbermudezme

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 5]
```diff
@@ -50,7 +57,7 @@ $main-font-size: 0.875em;
         margin: 0 14px 0 0;
 
         a {
-          color: $footer-contrast;
+          color: #003057,
```
Comment (ivanvgh) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Let's keep using color variables @anfbermudezme

**Review by ivanvgh — CHANGES_REQUESTED** [review]

**Review by ivanvgh — APPROVED** [review]

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 6]
```diff
@@ -3,6 +3,7 @@
 $line-color: rgba(255,255,255,0.2);
 $letter-spacing: 2.6px;
 $main-font-size: 0.875em;
+$serif: 'Open Sans';
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: This definition already exists in _variables.scss. @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 7]
```diff
@@ -50,7 +57,7 @@ $main-font-size: 0.875em;
         margin: 0 14px 0 0;
 
         a {
-          color: $footer-contrast;
+          color: $footer-background,
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Missing semicolon. @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 8]
```diff
@@ -104,27 +112,28 @@ $main-font-size: 0.875em;
...
+      line-height: 21px;
+    }
 
+    .copyright {
+      margin-top: 20px;
+      color: $footer-contrast;
+      text-transform: capitalize;
+      font-family: $serif;
+      font-size: $main-font-size;
+      line-height: 21px;
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss:128]: Add a blank line after this line. @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/templates/footer.html [hunk 9]
```diff
@@ -28,63 +28,78 @@
               <img alt="${_('organization logo')}" src="${footer['logo_image']}">
             </a>
           </p>
-          <h4>Empowering people to make progress in their lives through learning</h4>
+          <h4>Empowering people to make <br>
+            progress in their lives through <br>
+            learning</h4>
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/templates/footer.html]: Are the 'br' tags required? @anfbermudezme

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-pathways-theme/lms/templates/footer.html (hunk 9 above)
Comment (anfbermudezme) [reply edx-platform/pearson-pathways-theme/lms/templates/footer.html]: Yes because if we don't have de 'br' tags, the text looks like:

Selection_222

And in figma the text is required like:

Selection_223

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 10]
```diff
@@ -81,18 +87,19 @@ $main-font-size: 0.875em;
...
-      margin-top: 40px;
+      margin-top: 20px;
 
       li {
         display: inline-block;
         margin: 0 42px 0 0;
+        font-family: $font-family-sans-serif;
         font-size: $main-font-size;
-        letter-spacing: $letter-spacing;
+        line-height: 21px;
```
Comment (Squirrel18) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_footer.scss:101]: Why did you remove the 'letter spacing' rule? If you are not using it, please remove the variable as well. @anfbermudezme

File: edx-platform/pearson-pathways-theme/lms/templates/footer.html (hunk 9 above)
Comment (Squirrel18) [reply edx-platform/pearson-pathways-theme/lms/templates/footer.html]: And what happens on mobile screens? I think it's better to avoid pinning the elements and letting the elements adjust to their container. @anfbermudezme

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

**Review by Squirrel18 — APPROVED** [review]
